### DISCUSSION TITLE
Wifi modules(AC8265) use wpa_cli error.

### DISCUSSION THREAD

**Discussion by leadercxn** [OE4T/tegra-demo-distro#250] [General]
Hi all,
I use branch `kirkstone-l4t-r32.7.x` and add ` IMAGE_INSTALL += kernel-modules  linux-firmware ` for support my AC8265 wifi module
<img width="534" alt="image" src="https://user-images.githubusercontent.com/43691182/218712337-aa9f9247-a472-48cb-9ca8-67c3751655e2.png">

and I do these cmd like the followed picture, and the system occured `Failed to connect to non-global ctrl_ifname: wlP4p1s0  error: No such file or directory` and if I use `iwlist wlP4p1s0 scan` and it can scan the wifi ssid which around my environment
<img width="906" alt="image" src="https://user-images.githubusercontent.com/43691182/218713163-8d5c266a-261c-4c94-b382-8a7c5a735e69.png">

**Comment by ichergui** [comment]
Hey @leadercxn 

You need to set the country code before start scanning 
Something like: 

```
$  sudo iw reg set GB
```

**Reply by leadercxn** [reply]
@ichergui emm, still fail..
<img width="917" alt="image" src="https://user-images.githubusercontent.com/43691182/219286755-3bec2cc0-d732-40f2-86e0-62e2e2075126.png">

**Reply by leadercxn** [reply]
I can `iw dev  wlP4p1s0 scan` to scan  other wifi signals, just `wpa_cli` error...

**Reply by madisongh** [reply]
Unfortunately, we don't have any general-purpose configuration for setting up WiFi interfaces here, so you'll need to supply that for yourself.  That could be via config file(s) for wpa_supplicant, a systemd-networkd config file for the interface, and enabling/starting the systemd service. Alternatively, you could pull in NetworkManager or connman and configure those appropriately.

If you use the Sato GUI (pulled in from OE-Core via demo-image-sato or demo-image-full images), you should be able to configure WiFi interfaces through its control panel.  I believe it uses connman to manage the WiFi connections.
